python/lsst/ts/cRIOpy/GUI/EventWindow.py

```diff
@@ -0,0 +1,67 @@
+# This file is part of M1M3 SS GUI.
+#
+# Developed for the LSST Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+from PySide2.QtCore import Slot
+from PySide2.QtWidgets import QWidget, QVBoxLayout, QTreeView
+from PySide2.QtGui import QStandardItemModel, QStandardItem
+
+
+class EventWindow(QWidget):
```

Comment:
Please consider to improve the document of this class.

Reply:
Documented.